packages/regl-worldview/src/commands/Points.js

```diff
@@ -104,6 +104,8 @@ const makePointsCommand = (useWorldSpaceSize: boolean) => {
   };
 };
 
+export const points = makePointsCommand(false);
```

Comment:
Not letting the param be configurable might give us a headache in the future -- webviz uses it both ways. Maybe we should export `makePointsCommand` instead.

Reply:
Agreed. Check my updated description in this PR. I'm using two different approaches based on whether or not the command requires extra params.